### ISSUE TITLE
Fix focus trap on shift-TAB inside b2b table component

### ISSUE THREAD

**Issue by kacperknapik** [SAP/spartacus#9532]
### Current
Focus trap inside b2b table component while you shift-TAB. No focus trap for TAB only.

### Expected
Remove the focus trap - user should be able to go from the bottom to the top of the page. While inside the table, you should be able to shift-TAB to the elements above it.
